examples/node-speak-live/index.js

```diff
@@ -0,0 +1,44 @@
+const { createClient, LiveTTSEvents } = require("../../dist/main/index");
+const fetch = require("cross-fetch");
+
+const live = async () => {
+  const text = "Hello, how can I help you today?";
+
+  const deepgram = createClient(process.env.DEEPGRAM_API_KEY, {
+    global: { fetch: { options: { url: "https://api.beta.deepgram.com" } } },
+  });
+
+  const connection = deepgram.speak.live({ text }, { model: "aura-asteria-en" });
```

Comment:
```suggestion
  const connection = deepgram.speak.live({ model: "aura-asteria-en" });
```

The function definition for this expects `SpeakSchema, endpoint` - passing the `model` object as the second argument broke the function.